scripts/variantstore/scripts/merge_and_rescore_vdses.py

```diff
@@ -0,0 +1,243 @@
+import argparse
```

Comment:
Most of this is source from Chris's Echo-era PR [here](https://github.com/broadinstitute/gatk/pull/8867).